concord-server/server/src/test/java/com/rabidgremlin/concord/resources/LabelsResourceTest.java

```diff
@@ -18,58 +18,100 @@
 
 import com.rabidgremlin.concord.api.Label;
 import com.rabidgremlin.concord.auth.Caller;
-import com.rabidgremlin.concord.dao.LabelsDao;
 
 public class LabelsResourceTest
+    extends BaseResourceTest
 {
   private LabelsResource labelsResource;
 
-  private List<Label> labels = new ArrayList<>();
-
   @Mock
   private Caller callerMock;
 
-  @Mock
-  private LabelsDao labelsDaoMock;
-
   @Before
   public void setUp()
+    throws Exception
   {
     MockitoAnnotations.initMocks(this);
-    labelsResource = new LabelsResource(labelsDaoMock);
 
-    Label label1 = new Label();
-    label1.setLabel("OrderTaxi");
-    label1.setShortDescription("Order Taxi");
-    label1.setLongDescription("Where would you like your taxi to be sent ?");
+    setUpDatabase();
```

Comment:
Do we need an @after or @afterClass to tear down the DB?

Reply:
@joedai has a shutdown hook so it gets cleanly shutdown at the end. Sames some start up time for the tests.